external/loaders/loaders/mappers/_fine_res.py

```diff
@@ -41,24 +41,31 @@ def standardize_coords(
     return gfdl_to_standard(ds_shifted).drop("tile")
 
 
-def _open_merged_dataset(
-    fine_url: str, additional_dataset_urls: Optional[Sequence[str]]
-) -> FineResBudget:
-
-    fine = open_zarr(fine_url)
-    fine_shifted = standardize_coords(fine)
+def _merge(fine, additional_dataset_urls):
 
     if additional_dataset_urls is not None:
         additional_datasets = []
         for url in additional_dataset_urls:
             additional_datasets.append(open_zarr(url))
-        merged = xr.merge([fine_shifted, *additional_datasets], join="inner")
+        merged = xr.merge([fine, *additional_datasets], join="inner")
         if "latitude" in merged:
             merged["latitude"] = merged.latitude.isel(time=0)
         if "longitude" in merged:
             merged["longitude"] = merged.longitude.isel(time=0)
     else:
-        merged = fine_shifted
+        merged = fine
+
+    return merged
+
+
+def _open_merged_dataset(
+    fine_url: str, additional_dataset_urls: Optional[Sequence[str]]
+) -> FineResBudget:
```

Comment:
instead of splitting this function (and using a different one in each mapper), could we just have `standardize_fine_res_metadata` be a boolean kwarg for this func?

Reply:
Depends on answer to above I think.

Reply:
Done as discussed